README.md

```diff
@@ -26,22 +26,32 @@ The ```winetricks``` package should be used if it is available and up to date. T
 * Slackbuilds (Slackware): https://slackbuilds.org/apps/winetricks/
 * Ubuntu: https://packages.ubuntu.com/search?keywords=winetricks
 
-Note: packaged Debian / Ubuntu winetricks versions are typically outdated, so a manual installation is recommended.
 
 ## Manual Install
 
 If the package is unavailable, outdated, or the latest version is desired, a manual installation of winetricks can be done.
-It is _highly_ recommended to uninstall any previously installed version of winetricks first.
 
-**_If you don't uninstall a previously installed, packaged version of winetricks... Well then you get to pick up the pieces!_**
+### Quick install for one time usage
+
+To quickly grab current winetricks open a terminal and type
+~~~~~
+wget  https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks
```

Comment:
Ps. I never seen `~~~~` but I think ``` is normally used for a code block.

Like so:

```sh
echo "Hello world";
````

Or python highlighting:

```py
print("hello world");
```

Reply:
Thanks, I fixed the formatting.

There's an "Outdated" label in your comment. Not sure what that implies.

Reply:
That means, if you are unfamiliar with GitHub PRs, you committed a change this this file. Making the current code/markdown I see above outdated. Hence the reason GitHub marks it now as outdated. 

Ps. Which doesn't mean the code is actually fixed correctly, again only saying there is a newer version.

Reply:
Ah, thanks. So I marked this comment as resolved.